unity/acsShowcase/Assets/Scripts/Calling/CallScenario.cs

```diff
@@ -498,13 +512,21 @@ public void UnShareCamera()
     }
     
     /// <summary>
-    /// Add new participant to this call
+    /// Add new participant to this call. This functionality is not implemented.
     /// </summary>
     public RemoteParticipant AddParticipant(CallIdentifier personID)
     {
         if (CurrentCall != null)
         {
-            return CurrentCall?.AddParticipant(personID);
+            if (CurrentCall is TeamsCall teamsCall)
+            {
+                //return teamsCall?.AddParticipant(personID, new AddTeamsParticipantOptions(System.Environment.CurrentManagedThreadId.ToString()));
+            }
+            else if (CurrentCall is CommunicationCall acsCall)
+            {
+                //return acsCall?.AddParticipant(personID);
+            }
```

Comment:
Can probably specify the branches here, and have a signle `return null`  outside the if/else

Reply:
Also, I'm not a huge fan of putting in code (even commented code) that we haven't tested.   So i'd suggest just logging function call, and returning....but up to you

```cs
/// <summary>
/// Add new participant to this call. This functionality is not implemented.
/// </summary>
public RemoteParticipant AddParticipant(CallIdentifier personID)
{
    Log.Warning<CallScenario>("Add participant currently not implemented");
    return null;
}